### PR TITLE
Updated readme for a lower C++ version issue

### PR DESCRIPTION
**Pull request by TejoSaiSwaroop**
Tried in a different system , faced this issue. So just mentioning here so that they can test it with manual installing of the passlib and cryptography modules.
Great Project btw :)

### CODE REVIEW

**Comment by nightmare-tech** [comment]
Can you share the screenshot or exact code of the error? Can you recreate the error and show the steps to recreate it?

**Comment by TejoSaiSwaroop** [comment]
1. Clone the repo using "git clone https://github.com/nightmare-tech/PWD_Vault.git"
2. cd PWD_Vault
3. I wanted to try in the CLI so I didn't create the virtual env.
4. Run the pip install -r requirements.txt
5. After the command executed this is what I ended up with.
6. error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pandas
Failed to build pandas
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pandas)

6.1 The whole execution is extremely big.
6.2 Trying to rerun the  pip install -r requirements.txt results in this after the previous execution.
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
7.But after manually installing pandas and cryptography using 
"pip install pandas"
"pip install cryptography"
the python program is executing as it should be.

**Comment by nightmare-tech** [comment]
Thanks.. I'll merge your PR

**Review by nightmare-tech — CHANGES_REQUESTED** [review]
I think the sentence would be clearer if we revised it to say...
"If this results in an error, it might be due to a lower C++ version. Try installing the Python modules manually."

**Comment by TejoSaiSwaroop** [comment]
Hey just added those changes and committed to this PR, let me know if that's okay.

**Review by TejoSaiSwaroop — COMMENTED** [review]
Added it and committed

File: README.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ It is a very small repository. So, I didn't need another way to install it. And
     source bin/activate
     ```
 
- 2. **Install the necessary packages**
-
+ 2. **Install the necessary packages. If this results in error it might the case of a lower C++ Version.Prefer installing the python modules manually**
```
Comment (nightmare-tech) [inline README.md:39]: Why should the user prefer installing the modules manually? Then what's the point of the script? 
Please stick to what I suggested earlier and add it in a new line.

**Comment by TejoSaiSwaroop** [comment]
Is there anything else should I do or did I do smtg wrong? Kindly let me know.

**Review by nightmare-tech — APPROVED** [review]

**Pull request merged**